Puppetfile

```diff
@@ -37,7 +37,9 @@ mod "puppetlabs/haproxy", "3.0.1"
 ##################################################################################
 
 # Enable collection of Puppet api-endpoint metrics
-mod "puppetlabs/puppet_metrics_collector", "5.3.0"
+mod "puppetlabs/puppet_metrics_collector",
+    git: "https://github.com/puppetlabs/puppetlabs-pe_perf_control_repo",
```

Comment:
This should reference the pmc repo :)
```
git: "https://github.com/puppetlabs/puppetlabs-puppet_metrics_collector.git",
```

Reply:
https://media.giphy.com/media/3og0INyCmHlNylks9O/giphy.gif

Reply:
how do you inline a gif?   Anyway... thanks for catching that.  Rather embarrassing.